README.md

```diff
@@ -16,20 +16,20 @@ The files *scalardl-web-client-sdk* and *scalardl-node-client-sdk* are based on
 After modifying the implementation of scalardl-javascript-sdk-base, we have to upgrade the version field in the  package.json. It looks like this:
 
 ```
-"version": "1.3.1",
+"version": "3.0.0",
```

Comment:
I think it is correct to have `3.0.0` since APIs are changed, but no one really uses it in production yet so let's make it 2.2.0 so that major version is consistent throughput the SDKs.